src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url = jdbc:mysql://localhost:3307/studentdatabase?useSSL=false
```

Comment:
maintain one format. should be property=value.

I can see how each line has a different level of spacing. Not ideal.